packages/components-css/card/index.scss

```diff
@@ -0,0 +1,22 @@
+.frameless-card {
+  --utrecht-heading-2-font-size: var(--frameless-card-font-size);
+  --utrecht-heading-3-font-size: var(--frameless-card-font-size);
+  --utrecht-heading-4-font-size: var(--frameless-card-font-size);
+  --utrecht-heading-5-font-size: var(--frameless-card-font-size);
+  --utrecht-heading-6-font-size: var(--frameless-card-font-size);
+  --utrecht-heading-2-margin-block-end: var(--frameless-card-margin-block-end);
+  --utrecht-heading-3-margin-block-end: var(--frameless-card-margin-block-end);
+  --utrecht-heading-4-margin-block-end: var(--frameless-card-margin-block-end);
+  --utrecht-heading-5-margin-block-end: var(--frameless-card-margin-block-end);
+  --utrecht-heading-6-margin-block-end: var(--frameless-card-margin-block-end);
+  --utrecht-space-around: 1;
```

Comment:
Is deze nodig?

Reply:
In de Utrecht-stijlen wordt de margin-block-end van een heading berekend met behulp van de `calc` functie, waarbij de variabele `--utrecht-space-around` wordt gebruikt. Standaard staat deze variabele op 0, waardoor alle berekende marges ook 0 zijn. En dit overschrijft mijn eigen styling.

Om dit probleem op te lossen, heb ik `--utrecht-space-around` ingesteld op 1. Hierdoor wordt elke berekening met deze variabele, zoals `calc(var(--utrecht-space-around) * 10px)`, vermenigvuldigd met 1. Bijvoorbeeld, `calc(1 * 10px)` blijft 10px, en zo wordt je eigen styling behouden zonder overschreven te worden.